flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/connectors/hive/HiveDialectQueryITCase.java

```diff
@@ -330,6 +330,22 @@ public void testJoinInvolvingComplexType() throws Exception {
         }
     }
 
+    @Test
+    public void testCurrentDatabase() {
+        List<Row> result =
+                CollectionUtil.iteratorToList(
+                        tableEnv.executeSql("select current_database()").collect());
+        assertThat(result.toString()).isEqualTo("[+I[default]]");
+        tableEnv.executeSql("create database db1");
+        tableEnv.executeSql("use db1");
+        result =
+                CollectionUtil.iteratorToList(
+                        tableEnv.executeSql("select current_database()").collect());
+        assertThat(result.toString()).isEqualTo("[+I[db1]]");
+        // switch to default database for following test use default database
+        tableEnv.executeSql("use default");
```

Comment:
It's better if we can use 

```
@After
public void cleanup() {
            tableEnv.executeSql("use default");
}

```

Reply:
I think we don't need to add a `cleanup` method for currently only one test needs it. It would be better to limit it in this single test.